src/plugins/plugin.tooltip.js

```diff
@@ -1141,7 +1141,7 @@ export default {
         tooltip
       };
 
-      if (chart.notifyPlugins('beforeTooltipDraw', args) === false) {
```

Comment:
I think this should pass `{ ...args, cancelable: true }` otherwise the hook no longer receives the tooltip

Reply:
Ah yeah, fair.